docs/design/gang_scheduling.md

```diff
@@ -0,0 +1,631 @@
+---
+id: gang_scheduling
+title: Gang scheduling design
+---
+
+<!--
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ -->
+# Gang Scheduling Implementation
+## Change log
+
+| Change Date | Editors               | Description                                    |
+| ----------- | --------------------- | ---------------------------------------------- |
+| 2020-11-25  | Wilfred Spiegelenburg | Initial document                               |
+| 2020-11-26  | Wilfred Spiegelenburg | Allocation release definition, order change    |
+| 2020-11-27  | Wilfred Spiegelenburg | Sequence diagram                               |
+| 2020-12-12  | Wilfred Spiegelenburg | Application submit and cleanup                 |
+| 2020-12-15  | Wilfred Spiegelenburg | AllocationRelease message and cleanup sequence |
+| 2020-12-24  | Wilfred Spiegelenburg | Allocation changes for recovery                |
+| 2021-02-15  | Wilfred Spiegelenburg | Placeholder allocation and ask cleanup         |
```

Comment:
To keep the format consistency, can we remove this part?
we can link to the original google doc if people want to know more about the changes

Reply:
removed